wp-content/plugins/zippicks-social/includes/class-follow-manager.php

```
@@ -0,0 +1,645 @@
+    
+    /**
+     * Update follow statistics
+     *
+     * @param int $entity_id
+     * @param string $entity_type
+     * @return void
+     */
+    private function update_follow_stats(int $entity_id, string $entity_type): void {
+        global $wpdb;
+        $stats_table = $wpdb->prefix . 'zippicks_follow_stats';
+        $follows_table = $wpdb->prefix . 'zippicks_follows';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$follows_table'. (Unused Code Rules)

(UnusedLocalVariable)